go/oasis-test-runner/scenario/e2e/e2e.go

```diff
@@ -0,0 +1,147 @@
+// Package e2e implements the Oasis e2e test scenarios.
+package e2e
+
+import (
+	"github.com/oasislabs/oasis-core/go/oasis-test-runner/cmd"
+	"github.com/oasislabs/oasis-core/go/oasis-test-runner/scenario"
+	flag "github.com/spf13/pflag"
+	"math"
+
+	"github.com/oasislabs/oasis-core/go/common/logging"
+	"github.com/oasislabs/oasis-core/go/oasis-test-runner/env"
+	"github.com/oasislabs/oasis-core/go/oasis-test-runner/oasis"
+)
+
+const (
+	cfgNodeBinary = "node.binary"
+	cfgHaltEpoch  = "halt_epoch"
```

Comment:
This does not appear to be used anywhere in the test runner, can you check and remove in this case?

Reply:
It's used in the new `e2e.Parameters()` ;)
https://github.com/oasislabs/oasis-core/blob/matevz/feature/bench/go/oasis-test-runner/scenario/e2e/e2e.go#L65-L66

Reply:
And where is `sc.haltEpoch` used?

Reply:
Removed.